src/item.h

```diff
@@ -967,6 +967,9 @@ class Item : virtual public Thing
 			}
 		}
 		uint32_t getDefaultDuration() const {
+			if (items[id].decayTimeMax) {
```

Comment:
This method gonna return random values now, how we gonna get durationmin and durationmax in this case? I think getDefaultDuration shouldn't be changed and getDefaultDurationMax should be added.